packages/general/src/util/Promises.ts

```diff
@@ -55,7 +56,7 @@ export function anyPromise<T>(promises: ((() => Promise<T>) | Promise<T>)[]): Pr
                 .catch(reason => {
                     numberRejected++;
                     if (!wasResolved && numberRejected === promises.length) {
-                        reject(reason);
+                        reject(asError(reason));
```

Comment:
For some of these low-level utility classes perhaps we should leave the error alone... So add ignore eslint rule...

My thinking is that a.) the caller will need to do this check anyway, b.) the stack will be more useful if the caller adds it, and c.) a library user that for some reason throws non-errors will not expect the type of the throw to change

Reply:
Alright. I then disable the eslint check in these cases.